modules/ROOT/pages/connected-apps-permissions.adoc

```diff
@@ -0,0 +1,69 @@
+= Connected Apps Permissions by User
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Connected Apps can be used by Organization Administrators, developers, and end-users; each of these roles has different privileges when connecting applications,
```

Comment:
```suggestion
Connected apps can be used by organization administrators, developers, and end-users; each of these roles has different privileges when connecting applications,
```
I'm not sure about organization administrators. That is an actual role so it might need to remain title case and be bolded.  Kim will know.